apps/prairielearn/src/ee/pages/instructorInstanceAdminLti13/instructorInstanceAdminLti13.ts

```diff
@@ -106,8 +106,6 @@ router.get(
       sql.select_assessments,
       {
         course_instance_id: res.locals.course_instance.id,
-        authz_data: res.locals.authz_data,
-        req_date: res.locals.req_date,
```

Comment:
This errored out because of the `errorOnUnusedParameters` option in `@prairielearn/postgres`.

This regressed in #13332. Those changes were not tested at the time.